drools-yaml-rules-efesto/drools-yaml-rules-api/src/main/java/org/drools/yaml/api/context/jpy/JpyDurableRulesEngine.java

```diff
@@ -14,8 +14,12 @@
 import java.util.Map;
 import java.util.function.Function;
 
-import static org.drools.yaml.core.RulesExecutor.OBJECT_MAPPER;
-
+/**
+ * This is the library that is being used in Python layer.
+ * If this class gets renamed, moved to another package,
+ * or you add/remove/rename methods, make sure to align the changes
+ * on the Python end too.
+ */
```

Comment:
please be aware that currently https://github.com/benthomasson/ansible-events/pull/136/files#diff-66a4a18d3561ada19693296ccfeec9b9929ff098147f6ce2da5388dd4ff950a0R17
the package name is 

```
package org.drools.yaml.durable.jpy;
```
this PR is changing it to:
```
package org.drools.yaml.api.context.jpy;
```
so make sure to reflect the change on the Python end